assets/tests/components/card.test.tsx

```diff
@@ -66,10 +68,29 @@ describe("Card", () => {
     )
 
     expect(
-      result.queryByText(/Foo/)?.parentElement?.parentElement?.className
+      result.queryByText(/Foo/)?.parentElement?.parentElement?.parentElement
+        ?.className
     ).toMatch(/m-card--no-focus-or-hover/)
   })
 
+  test("can mark as selected", () => {
+    const result = render(
+      <Card
+        title="My Card"
+        currentTime={new Date()}
+        style="kiwi"
+        selected={true}
+      >
+        Foo
+      </Card>
+    )
+
+    expect(
+      result.queryByText(/Foo/)?.parentElement?.parentElement?.parentElement
+        ?.className
+    ).toMatch(/selected/)
+  })
```

Comment:
Can we express this with either aria-current or aria-selected?
https://www.w3.org/TR/wai-aria-1.1/#aria-current

```
expect(screen.getByRole("generic" | "listitem", { name: title, current: true | selected: true })).toBeVisible()
```

if the card is always used in a list, I think it should be `aria-selected`
if it's not in a list I think it should be `aria-current`

And just for completeness, If there can be multiple selections in the future then I think we should add `aria-multiselect` to the container list

Reply:
Ah, good call! It will probably vary whether the card component is used in a list or not, so I'll have to do some thinking about how to best get the component to set the appropriate ARIA attribute.

Reply:
On the other hand I'm not sure if it really makes sense for a standalone card not in a list to be "selected", so maybe just using `aria-selected` is safe unless we extend `<Card />` to some as-yet-unforeseen use cases.

Reply:
Hmm, now that I read the documentation I think `aria-current` might actually be what we want? It looks like `aria-selected` is for some more specific roles like `tablist` that I don't think apply here, but let me know if I'm missing something.

Reply:
Okay I just pushed my changes (64a6dba9), they may be a little janky but I'm curious to hear what you think. I may or may not have spent like 15 minutes stuck on tests because of `aria-lebeledby` versus `aria-labelledby`... 🙈

Reply:
Well one of the nice things about React Testing Library is that if the test ended up looking good, there's a good chance that your HTML it decent as well. As for other improvements to the card, I think those are probably out of scope for this particular PR but definitely create a tech debt ticket with any ideas and we can talk about it!